core/src/main/java/nl/nn/adapterframework/management/bus/endpoints/Monitoring.java

```diff
@@ -280,17 +285,17 @@ private Map<String, Object> mapMonitor(Monitor monitor) {
 		monitorMap.put("name", monitor.getName());
 		monitorMap.put("type", monitor.getType());
 		monitorMap.put("destinations", monitor.getDestinationSet());
-		monitorMap.put("lastHit", monitor.getLastHit());
+		monitorMap.put("lastHit", monitor.getLastHit() != null ? monitor.getLastHit().toEpochMilli() : null);
```

Comment:
I think it's better to have a small conversion helper method for this, instead of repeating this.
Or checking if a custom mapping can be added to Jackson.

Reply:
Ik laat het aan nou over, ik heb het PR ge-approved zoals het nu is. :-)